docs/development/end-to-end-testing.md

```diff
@@ -34,3 +34,48 @@ You need to make sure to install the official [cucumber](https://marketplace.vis
 
 Then you will have all the autocomplete features available to you
 
+### Running e2e tests locally
+
+Running e2e tests locally unfortunately requires some extra steps to make it work with our current setup. This is because docker containers can't communicate with the host machine using localhost. To make this work we need to add an entry to the /etc/hosts file on your machine. This is only needed when running the tests locally. When running the tests in the pipeline this is not needed.
+
+#### Steps to Add host.docker.internal Entry to /etc/hosts File
+When working with Docker, adding host.docker.internal to your /etc/hosts file allows Docker containers to access services running on the host machine. Follow these steps to add this entry:
+
+1. Open the /etc/hosts File:
+   - You need administrative privileges to edit the /etc/hosts file.
+   - Open the file in a text editor of your choice. For example, using vim, you would use the following command:
+    ```bash
+        sudo vim /etc/hosts 
+    ```
+2. Add the host.docker.internal Entry:
+   - In the /etc/hosts file, add a new line to link host.docker.internal to the IP address of your host machine. This is typically 127.0.0.1 (localhost).
+   - The entry should look like this:
+    ```csharp
+        # ZAC
+        127.0.0.1 host.docker.internal
+    ```
+   - Save the file and exit the text editor.
+3. Verify the Entry:
```

Comment:
This verification is not really needed I would say but oh well. I mean: it's just an entry in your hosts file. Nothing special at all.

Reply:
Ok clear. But just to verify a change in your host file, surely it is not needed to spin up an entire Docker container? Why not just do a `ping host.docker.internal` on your command line?

Reply:
Never mind: with Docker I guess you are testing if you can access it from inside a running Docker container so I guess that is indeed nice. :-)